bazel/init/stage_1.bzl

```diff
@@ -143,7 +143,9 @@ def stage_1():
         sha256 = "51d676b6846440210da48899e4df618a357e6e44ecde7106f1e44ea16ae8adc7",
         strip_prefix = "abseil-cpp-20230125.3",
         patch_args = ["-p1"],
-        patches = ["@enkit//bazel/dependencies/abseil:0001-absl-flags-parse.cc-provide-a-mechanism-to-let-other.patch"],
+        patches = ["@enkit//bazel/dependencies/abseil:0001-absl-flags-parse.cc-provide-a-mechanism-to-let-other.patch",
```

Comment:
nit: indentation is wrong, maybe run fix_format.sh from internal on this file? if not,
```
patches = [
    "...",
    "...",
]

Reply:
Done.